packages/market-service/src/osmosis/osmosis.ts

```diff
@@ -49,6 +50,30 @@ export class OsmosisMarketService implements MarketService {
     if (!adapters.assetIdToOsmosis(assetId)) return null
 
     try {
+      const assetReference = fromAssetId(assetId).assetReference
+      if (isOsmosisLpAsset(assetReference)) {
+        /* No market exists for Osmosis pool assets, but we can calculate the 'price' of each pool token
```

Comment:
```suggestion
        /* No market data exists for Osmosis pool assets, but we can calculate the 'price' of each pool token
```

Reply:
This isn't a typo; there is nowhere to buy/sell Osmosis LP tokens.